types/d3-drag/index.d.ts

```diff
@@ -91,7 +91,7 @@ export interface DragBehavior<GElement extends DraggedElementBaseType, Datum, Su
      * in order, being passed the current datum (d), the current index (i), and the current group (nodes),
      * with this as the current DOM element. The function returns the container element.
      */
-    container(accessor: ValueFn<GElement, Datum, DragContainerElement>): this;
+    container(accessor: ValueFn<GElement, Datum, DragContainerElement | null>): this;
```

Comment:
As always, open to clarifying motivation. But I meant to submit the original review as "asking for changes", rather than just "comment".

Reply:
you are right, this was the wrong change. not sure why i have done that in the first place now. The issue is that `SVGElement.ownerSVGElement` is now defined to be nullable, which seems to be accurate per the spec, https://svgwg.org/svg2-draft/types.html#__svg__SVGElement__ownerSVGElement, so we need a null assertion in the tests instead of changing the definition of the function. Should be fixed now.